tests/test_rat_board.py

```diff
@@ -280,8 +280,9 @@ def test_search_valid(self, test_input, rat_board_fx: RatBoard, rescue_plain_fx:
                             True,
                             3.14,
                             '@12345678-9876-53d1-ea5e-0000dead',
-                            '@12345678-9876-53d1-ea5e-000deadsheep'
-                            "42",
+                            '@12345678-9876-53d1-ea5e-0000deadbee-',
+                            '@12345678-9876-53d1-ea5e-000deadsheep',
```

Comment:
this is not a valid uuid as `s` is not within the range of hexidecimal.

Reply:
This ensures the exception is hit as it's the proper length (37 characters) but not a valid UUID. Mostly for user error since this function will take user input. Otherwise I didn't get 100% coverage ;)